packages/atomic/src/components/ipx/atomic-ipx-result-link/atomic-ipx-result-link.tsx

```diff
@@ -0,0 +1,127 @@
+import {isUndefined} from '@coveo/bueno';
+import {
+  GenericAnalyticsActionCreators,
+  loadGenericAnalyticsActions,
+} from '@coveo/headless';
+import {Component, h, Prop, Element} from '@stencil/core';
+import {Bindings} from '../../../components';
+import {buildCustomEvent} from '../../../utils/event-utils';
+import {
+  InitializableComponent,
+  InitializeBindings,
+} from '../../../utils/initialization-utils';
+import {buildStringTemplateFromResult} from '../../../utils/result-utils';
+import {getDefaultSlotFromHost} from '../../../utils/slot-utils';
+import {
+  AnyInteractiveResult,
+  AnyUnfoldedResult,
+} from '../../common/interface/result';
+import {getAttributesFromLinkSlot} from '../../common/result-link/attributes-slot';
+import {LinkWithResultAnalytics} from '../../common/result-link/result-link';
+import {
+  ResultContext,
+  InteractiveResultContext,
+} from '../../search/result-template-components/result-template-decorators';
+
+/**
+ * The `atomic-ipx-result-link` component automatically transforms a search result title into a clickable link that points to the original item. It is an experimental internal component not intended for general use.
+ * @slot default - Lets you display alternative content inside the link
+ * @slot attributes - Lets you pass [attributes](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#attributes) down to the link element, overriding other attributes, to be used exclusively with an "a" tag such as `<a slot="attributes" target="_blank" download></a>`.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-ipx-result-link',
+  styleUrl: 'atomic-ipx-result-link.pcss',
+  shadow: false,
+})
+export class AtomicIPXResultLink implements InitializableComponent {
```

Comment:
Ont peux tu hérité de result link direct?

Reply:
No Inheritance for components is our rule of thumb. Do prefer composition :)